src/w2toolbar.js

```diff
@@ -362,6 +362,14 @@
                 el.html(html);
                 if (it.hidden) { el.css('display', 'none'); } else { el.css('display', ''); }
                 if (it.disabled) { el.addClass('disabled'); } else { el.removeClass('disabled'); }
+                if (it.type == 'check' || it.type == 'drop' || it.type == 'menu') {
```

Comment:
checked class is already added in line 443 (getItemHTML method). I do not see why you need it here too.